package.json

```diff
@@ -56,10 +60,6 @@
   "engines": {
     "node": ">=6"
   },
-  "files": [
-    "bin",
-    "dist"
```

Comment:
I've removed this restrictions to deploy all files necessary for the postinstall build.